src/components/SingleReview.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const SingleCategory = (props) => {
+    const [review, setReview] = useState([]);
+    const { id } = useParams();
+    useEffect(() => {
+        fetch(`https://pacific-citadel-03998.herokuapp.com/api/reviews/${id}`)
```

Comment:
I'd also suggest placing all of you request functions in one api.js file, and then importing in each one as you need it.